src/utility/include/mp-units/math.h

```diff
@@ -136,6 +136,27 @@ template<auto R, typename Rep>
   return {static_cast<Rep>(abs(q.numerical_value_ref_in(q.unit))), R};
 }
 
+/**
+ * @brief Computes the fma of 3 quantities
+ *
+ * @param a: Multiplicand
+ * @param x: Multiplicand
+ * @param b: Addend
+ * @return Quantity: The nearest floating point representable to ax+b
+ */
+template<auto R, auto S, typename Rep>
+[[nodiscard]] constexpr quantity<R, Rep> fma(const quantity<R, Rep>& a, const quantity<S, Rep>& x,
+                                             const quantity<R, Rep>& b) noexcept
+  requires requires { fma(a.numerical_value_ref_in(a.unit)); } ||
+           requires { std::fma(a.numerical_value_ref_in(a.unit)); }
```

Comment:
Shouldn't these have 3 arguments, just like the implementation?

Reply:
Yeah, it doesn't work, among other things: <https://cpp1.godbolt.org/z/E59bbv9b5>.

Reply:
Now there remains argument/parameter mismatches.

Reply:
See the comment above and the updated CE link.
The tests in the PR don't compile.